traitsui/examples/demo/Useful/index.rst

```diff
@@ -0,0 +1 @@
+These demonstrations show off some useful features of TraitsUI.
```

Comment:
Additionally, i'm leaning towards renaming the directory `Misc` or something on those lines instead of `Useful`

Reply:
> Additionally, i'm leaning towards renaming the directory `Misc` or something on those lines instead of `Useful`

I tried to make this change but the CI was failing on ` test_all_examples` saying `/home/travis/.edm/envs/traitsui-test-3.6-pyqt5/lib/python3.6/site-packages/traitsui/examples/demo/Useful/demo_group_size.py does not exist.` but I could not find the test_all_examples to make the update? I reverted it for now, but it could probably very easily be switched